Documentation/network/bgp-control-plane.rst

```diff
@@ -337,69 +337,87 @@ a BGP control plane.
 
 -  evaluate the desired ``CiliumBGPPeeringPolicy``
 -  create/remove the desired BGP routers
 -  advertise/withdraw the desired BGP routes
 -  enable/disable any BGP server specific features
 -  inform the caller if the policy cannot be applied
 
-The GoBGP implementation is capable of evaluating each ``CiliumBGPVirtualRouter`` in isolation. This
-means when applying a ``CiliumBGPPeeringPolicy`` the GoBGP ``BGPRouterManager`` will attempt to
-create each ``CiliumBGPVirtualRouter``. If a particular ``CiliumBGPVirtualRouter`` fails to
-instantiate the error is logged and the ``BGPRouterManager`` will continue to the next
-``CiliumBGPVirtualRouter``, utilizing the aforementioned logic.
+The ``BGP Manager`` evaluates each ``CiliumBGPVirtualRouter`` in isolation. 
+While applying a ``CiliumBGPPeeringPolicy``, it will attempt to create each 
+``CiliumBGPVirtualRouter``.
+
+If a particular ``CiliumBGPVirtualRouter`` fails to instantiate, the error 
+message is logged, and the ``BGP Manager`` will continue to the next 
+``CiliumBGPVirtualRouter``.
+
+It is worth expanding on how the ``BGP Manager`` works internally.  
+``BGP Manager`` views each ``CiliumBGPVirtualRouter`` as a BGP router instance. 
+Each ``CiliumBGPVirtualRouter`` is defined by a local ASN, a router ID and a 
+list of ``CiliumBGPNeighbors`` with whom it will establish peering.
 
-GoBGP BGPRouterManager Architecture
-***********************************
+This is enough for the ``BGP Manager`` to create a ``BGP Router`` instance. 
+``BGP Manager`` groups ``Bgp Router`` instances by their local ASNs. 
 
-It's worth expanding on how the ``gobgp`` implementation of the
-``BGPRouterManager`` works internally. This ``BGPRouterManager`` views each
-``CiliumBGPVirtualRouter`` as a BGP router instance. Each ``CiliumBGPVirtualRouter`` defines a local
-ASN, a router ID and a list of ``CiliumBGPNeighbors`` to peer with. This is enough for the
-``BGPRouterManager`` to create a ``BgpServer`` instance, which is the nomenclature defining a BGP
-speaker in ``gobgp``-package-parlance. This ``BGPRouterManager`` groups ``BgpServer`` instances by
-their local ASNs. This leads to the following rule: A ``CiliumBGPPeeringPolicy`` applying to node
-``A`` must not have two or more ``CiliumBGPVirtualRouters`` with the same ``localASN`` fields.
+.. note::
+
+   A ``CiliumBGPPeeringPolicy`` applying to a node must not have two or more
+   ``CiliumBGPVirtualRouters`` with the same ``localASN`` fields.
 
-The ``gobgp`` ``BGPRouterManager`` employs a set of ``ConfigReconcilerFunc``\ (s) which perform the
-order-dependent reconciliation actions for each ``BgpServer`` it must reconcile. A
-``ConfigReconcilerFunc`` is simply a function with a typed signature.
+The ``BGP Manager`` employs a set of ``ConfigReconcilerFunc``\ (s) performing an 
+order-dependent reconciliation action for each ``BGP Router`` it must reconcile. 
+A ``ConfigReconcilerFunc`` is a  function with a typed signature.
```

Comment:
```suggestion
A ``ConfigReconcilerFunc`` is a function with a typed signature.
```

What does this info bring, by the way? What type is it?

Reply:
Hmm, it was already present in the document. I do agree that it does not bring much info in the documentation. I will drop this entirely.

Reply:
Replaced it with `Reconcilers`, which is a more accurate description of what is going on.